_posts/2019-09-23-exposing-http-bridge.md

```diff
@@ -0,0 +1,308 @@
+---
+layout: post
+title:  "Exposing your Apache Kafka cluster through the HTTP bridge"
+date: 2019-10-18
+author: paolo_patierno
+---
+
+In the previous blog [post](2019-07-22-http-bridge-intro.md), we introduced the new Strimzi HTTP bridge component.
```

Comment:
Should we say _in the previoud blog post about HTTP Bridge_? Because we had a lot of posts since